### ISSUE TITLE
hi

### ISSUE THREAD

**Issue by okeanosthedev** [Erebus009/crypto-news-tracker#58]
not an issue but i think we can add all the coins with uuid in https://rapidapi.com/Coinranking/api/coinranking1/ at index coins index
we can view all the coins uuid and names in there i think that (https://github.com/Erebus009/crypto-news-tracker/projects/2#card-70362448 can be made because the api gives the shorted name of the coin too)
we can take all coin i am not good at the api thing because of that my code didnt work but i tried this https://cdn.discordapp.com/attachments/707926768950050846/938894285602979910/unknown.png
(At all of this i can Make you a dark mode slider)

**Comment by denene12** [comment]
and i am using the template of this if i change the colors and gui will there be a issue

**Comment by Erebus009** [comment]
That would be a way to grab the UUID, the return function area was mostly to make it easier to read when we appended stuff to the page using JavaScript.  If we could map over the data beforehand and push the name of the coin and UUID into the coin list array it would be able to dynamically search coins instead of the hardcoded way that the search list is done now.

 Or easier pass the UUID into the fetch request itself.

**Comment by denene12** [comment]
İ say if we make search from api with name it will add the all coins at the api

**Comment by denene12** [comment]
And i can make your wish come true i can make you a dark mode slider its easy actually

**Comment by denene12** [comment]
![Uploading E3B6B41E-014A-4C55-A636-959D388FEF4D.jpeg…]()

**Comment by denene12** [comment]
![5B36CFB1-4DD2-4A90-A7C9-6A5C127BC537](https://user-images.githubusercontent.com/66648881/152651092-d7ccc75b-f1b1-4b52-999c-cc8b4aa20819.jpeg)

**Comment by Erebus009** [comment]
I was able to add it in after a few hours. Most of the issue was trying to get everything to work together. 

![image](https://user-images.githubusercontent.com/12164234/152656848-83f6756e-c99a-4956-8f39-59f694c99905.png)

Yeah, I'm cool if you can fork and add dark mode it's an open project

**Comment by denene12** [comment]
Yeah actually i am using the template in cryptoanalizer.tk i will change it but now i have some work i dont have much time but i will change its css would be a problem

**Comment by denene12** [comment]
![919418AA-97DD-45F1-A5EB-DE7C453F343D](https://user-images.githubusercontent.com/66648881/152657141-151a4531-cfa5-4657-970b-40022f41c4e5.jpeg)
And i wanna ask how you know what to write in www.xxxxxx.com/c͟o͟i͟n͟s͟?r͟e͟f͟e͟r͟a͟n͟c͟e͟.   The underlined part how you know what to write where do you look

**Comment by Erebus009** [comment]
just query parameters added together went to the full docs for that was curious what they had. 

![image](https://user-images.githubusercontent.com/12164234/152657539-1a643e0f-fca9-41e6-b9b3-cecf9c0e3de8.png)

**Comment by okeanosthedev** [comment]
Thanks to teaching me that maybe dark mode will finish today but i think it would finish tomorrow

**Comment by denene12** [comment]
and i need an dark  mode css can you. Make a new css that is dark

**Comment by denene12** [comment]
https://cdn.discordapp.com/attachments/685788050558943274/939820932057403432/unknown.png where can i change the css for the centered line

**Comment by Erebus009** [comment]
.table>tbody>tr>td, .table>tbody>tr>th,
.table>tfoot>tr>td, .table>tfoot>tr>th,
.table>thead>tr>td, .table>thead>tr>th
{
  border-bottom: red 1px solid;
}

**Comment by okeanosthedev** [comment]
İt is not at the style.css

**Comment by okeanosthedev** [comment]
Found it sorry

**Comment by okeanosthedev** [comment]
my man are you still alive you don't have any actions for a long time
